scripts/docker/docker-entrypoint.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -e
+
+cmd="$(basename "${1}")"
```

Comment:
If you are not familiar with docker, the ENTRYPOINT and CMD fields from the Dockerfile get concatenated together to form the command that gets run by default in the container. This script is the ENTRYPOINT, and it receives the fields of CMD as arguments. If the first argument to our ENTRYPOINT script is `swapd`, we make sure the file permissions are good before using `gosu` to drop root permissions and execute `swapd` as the `atomic` user.